Tasks/PublishBuildArtifacts/task.json

```diff
@@ -9,9 +9,10 @@
     "Build"
   ],
   "author": "Microsoft Corporation",
+  "preview": true,
   "version": {
-    "Major": 1,
-    "Minor": 131,
+    "Major": 2,
```

Comment:
we should take this opportunity to rename the task "PublishDropArtifact" since this should work in the future from a release as well